crates/ra_assists/Cargo.toml

```diff
@@ -21,4 +21,5 @@ ra_prof = { path = "../ra_prof" }
 ra_db = { path = "../ra_db" }
 ra_ide_db = { path = "../ra_ide_db" }
 hir = { path = "../ra_hir", package = "ra_hir" }
+hir_def = { path = "../ra_hir_def", package = "ra_hir_def" }
```

Comment:
ide crates should only depend on `hir` directly, `hir_xxx` is sort of private stuff for compiler internals.